src/Shared/PackageManagers/ProjectManagerFactory.cs

```diff
@@ -10,27 +10,6 @@ namespace Microsoft.CST.OpenSource.PackageManagers
 
     public static class ProjectManagerFactory
```

Comment:
Are these changes to the factory necessary for this PR, or is it just general cleanup of unused static methods?

Reply:
Yeah, the first one was being used but the second was unused. Now they're both unused so I removed them both.